Formula/iproute2mac.rb

```diff
@@ -1,11 +1,12 @@
 class Iproute2mac < Formula
   desc "CLI wrapper for basic network utilities on macOS - ip command"
   homepage "https://github.com/brona/iproute2mac"
-  url "https://github.com/brona/iproute2mac/releases/download/v1.2.3/iproute2mac-1.2.3.tar.gz"
-  sha256 "95ef8d4b0e32e4d3e3d975afa11d7aa0797d59b0f2c21b73a4e26d357bd6f93f"
+  url "https://github.com/brona/iproute2mac/releases/download/v1.3.0/iproute2mac-1.3.0.tar.gz"
+  sha256 "3fefce6b0f5e166355fdb04934cbdd906211b64e5adb6a385469696dc51233b7"
   license "MIT"
 
   bottle :unneeded
```

Comment:
```suggestion
```

Reply:
@chenrui333 What's the reasoning for this?

Reply:
The reasoning is it would depend on the `python@3.8`, which would naturally qualify the formula to have bottles.

Reply:
But I could be wrong on the `bottle` usage.

Reply:
Bottles probably aren't needed since this just installs a python script (for example: [`nicovideo-dl`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/nicovideo-dl.rb))